test/test_torch.py

```diff
@@ -4117,9 +4117,18 @@ def __reduce__(self):
                 self.assertEqual(x.size(), load_x.size())
                 self.assertEqual(x.stride(), load_x.stride())
 
+
+    def test_serialization_save_warnings(self):
+        with warnings.catch_warnings(record=True) as warns:
+            with tempfile.NamedTemporaryFile() as checkpoint:
+                x = torch.save(torch.nn.Module(), checkpoint)
```

Comment:
I'm not sure that one is supposed to instantiate a nn.Module() directly like this. Could we change this to `nn.Linear(2, 3)` to better reflect how a user would use the API?